src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsInt, IsBoolean } from 'class-validator';
+
+export class CreateUserDto {
```

Comment:
DTO must implement interface 
it will be 
"export class CreateUserDto implements ICreateUserInput"